README.md

```diff
@@ -2032,6 +2032,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [Killer Coda](https://killercoda.com/)  - Interactive playground in your browser to study Linux, Kubernetes, Containers, Programming, DevOps, Networking
   * [Axonomy App](https://axonomy-app.com/) - A free tool to create, manage and share your invoices with your clients. 10 free invoices per month.
   * [Table Format Converter](https://www.tableformatconverter.com) - A free tool to convert table data to different formats, such as CSV, HTML, JSON, Markdown and more.
+  * [Fashion Saled Daily API](https://chicsaledaily.com) - Discover the best fashion deals and discounts from thousands of brands and stores.
```

Comment:
Thanks for the contribution, I dont think this fits the purpose of the list, sorry